From be1d72887dada33a0f16b906bc9c549923d7d372 Mon Sep 17 00:00:00 2001
From: Finn Behrens <github@kloenk.de>
Date: Wed, 8 Nov 2023 20:25:12 +0100
Subject: [PATCH] config: also read from /etc/helix

---
 helix-loader/src/config.rs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/helix-loader/src/config.rs b/helix-loader/src/config.rs
index d092d20f73a8..9d96ec084ab2 100644
--- a/helix-loader/src/config.rs
+++ b/helix-loader/src/config.rs
@@ -10,6 +10,7 @@ pub fn default_lang_config() -> toml::Value {
 /// User configured languages.toml file, merged with the default config.
 pub fn user_lang_config() -> Result<toml::Value, toml::de::Error> {
     let config = [
+        std::path::PathBuf::from("/etc/helix"),
         crate::config_dir(),
         crate::find_workspace().0.join(".helix"),
     ]