diff --git a/backend/app/config.py b/backend/app/config.py
index 5dfd6b9..bce1bae 100644
--- a/backend/app/config.py
+++ b/backend/app/config.py
@@ -22,7 +22,6 @@ from oic.oic import Client
 from oic.oic.message import RegistrationResponse
 from oic.utils.authn.client import CLIENT_AUTHN_METHOD
 from flask import Flask, jsonify, request
-from flask_basicauth import BasicAuth
 from flask_migrate import Migrate
 from flask_sqlalchemy import SQLAlchemy
 from flask_bcrypt import Bcrypt
@@ -119,10 +118,6 @@ app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 app.config["JWT_SECRET_KEY"] = os.getenv("JWT_SECRET_KEY", "super-secret")
 app.config["JWT_ACCESS_TOKEN_EXPIRES"] = JWT_ACCESS_TOKEN_EXPIRES
 app.config["JWT_REFRESH_TOKEN_EXPIRES"] = JWT_REFRESH_TOKEN_EXPIRES
-if COLLECT_METRICS:
-    # BASIC_AUTH
-    app.config["BASIC_AUTH_USERNAME"] = os.getenv("METRICS_USER", "kitchenowl")
-    app.config["BASIC_AUTH_PASSWORD"] = os.getenv("METRICS_PASSWORD", "ZqQtidgC5n3YXb")
 
 convention = {
     "ix": "ix_%(column_0_label)s",
@@ -208,14 +203,12 @@ if FRONT_URL:
 
 
 if COLLECT_METRICS:
-    basic_auth = BasicAuth(app)
     registry = CollectorRegistry()
     multiprocess.MultiProcessCollector(registry, path="/tmp")
     metrics = PrometheusMetrics(
         app,
         registry=registry,
         path="/metrics/",
-        metrics_decorator=basic_auth.required,
         group_by="endpoint",
     )
     metrics.info("app_info", "Application info", version=BACKEND_VERSION)
